test/integration/NFTs721Subscription.test.ts

```diff
@@ -149,4 +149,15 @@ describe('Subscription NFTs (ERC-721) e2e Testing', () => {
 
   })  
 
+  test('As NFT Publisher I can whitelist to an account', async () => {
+    
+    const whitelistCommand = `${baseCommands.nfts721.mintSubscription} "${nftAddress}" --receiver ${execOpts.accounts[2]} --tokenId 1`
+    console.debug(`COMMAND: ${whitelistCommand}`)
```

Comment:
Not sure how the test are logging this, but better use the logger if possible

Reply:
in the testing we are using console everywhere